tests/mock_network/mod.rs

```diff
@@ -443,11 +443,11 @@ fn check_section_info_ack() {
     let nodes = create_connected_nodes_until_split(&network, vec![1, 1], true);
 
     // Due to the behaviour in the `create nodes until split`, there shall only be one section nodes
-    // sending out `section_info_ack`. Hence only one section shall receive the ack.
-    let num_of_receives = nodes
+    // sending out `section_info_ack`. Hence only one section shall consensus on it eventually.
+    let num_of_consensus = nodes
         .iter()
-        .filter(|node| node.inner.received_section_info_ack())
+        .filter(|node| node.inner.get_version() > min_section_size as u64)
```

Comment:
I am a tiny bit confused about that one, not clear why the version is related to min_section_size.
Are you trying to get any version that it not 0? Or with the pass through just checking if a version exists (i.e iter not empty may be enough?)

Reply:
yes, I `non-empty` check shall be enough here for this test.